src/redemptions/redemption.service.ts

```diff
@@ -78,63 +87,99 @@ export class RedemptionService {
         idDetails,
       };
     }
+    const repeatedFaceWorkflowIds = this.getRepeatedFaceWorkflowIds(
+      transactionStatus.capabilities.imageChecks,
+    );
+    const multiAccountFailure = await this.multiAccountFailure(
+      userId,
+      repeatedFaceWorkflowIds,
+    );
+
     // TODO: HANDLE WARN, use decision.risk.score?
-    const failure =
-      this.livenessStatus(transactionStatus) ||
-      this.similarityStatus(transactionStatus) ||
-      this.dataChecksStatus(transactionStatus) ||
-      this.extractionStatus(transactionStatus) ||
-      this.usabilityStatus(transactionStatus);
+    const failure = multiAccountFailure || this.labelFailure(labels);
     if (failure) {
       return {
         status: KycStatus.FAILED,
         failureMessage: failure,
         idDetails,
       };
     }
-
+    if (!multiAccountFailure && this.hasOnlyDuplicateFaceFailures(labels)) {
```

Comment:
think the `!multiAccountFailure` is spurious due to preceding block

Reply:
See this comment:
https://github.com/iron-fish/ironfish-api/pull/1323#discussion_r1128222672

Reply:
fair